Core/Executor/UserGroupManager.php

```diff
@@ -31,14 +31,17 @@ protected function create()
 
         $parentGroupId = $this->dsl['parent_group_id'];
         $parentGroupId = $this->referenceResolver->resolveReference($parentGroupId);
-
-        $parentGroup = $userService->loadUserGroup($parentGroupId);
+        $parentGroup = $this->userGroupMatcher->matchOneByKey($parentGroupId);
```

Comment:
This change is not harmful, but it does not add a lot of value either, as we still have to use the userService in this method, and it is a bit slower...